applications/RomApplication/custom_python/add_custom_strategies_to_python.h

```diff
@@ -24,6 +24,8 @@
 // Project includes
 #include "includes/define_python.h"
 #include "custom_strategies/rom_builder_and_solver.h"
+#include "custom_strategies/lspg_rom_builder_and_solver.h"
+#include "custom_strategies/petrov_galerkin_rom_builder_and_solver.h"
```

Comment:
I think you can add this two to the `add_custom_strategies_to_python.cpp` source file.

Reply:
Also adding `#include "custom_strategies/rom_builder_and_solver.h"`.